src/main/java/org/mule/extension/db/internal/result/statement/StatementResultIterator.java

```diff
@@ -264,4 +276,21 @@ protected SingleStatementResult doProcessResultSet(String name, ResultSet result
   public void remove() {
     throw new UnsupportedOperationException();
   }
+
+  private TypedValue<Object> handleClobType(Clob value) throws SQLException {
```

Comment:
This code is duplicated from the class InsensitiveMapRowHandler, can we do anything to avoid duplication?
Also, in the already existing handleClobType method it does not invoke the beginStreaming() method, should it be called?